include/picongpu/plugins/IsaacPlugin.hpp

```diff
@@ -366,7 +365,7 @@ class IsaacPlugin : public ILightweightPlugin
         {
             //using an internal variable "render_interval" as notifyPeroid
             //of PIConGPU cannot be changed at runtime
-            notifyPeriod = 1;
+            notifyPeriod = "1";
```

Comment:
@theZiz can you confirm this is correct for the ISAAC plugin?
The GUI can change the `render_interval`, right? This is not the same as the `notifyPeriod` I guess and should still work, no? (see: `//using an internal variable "render_interval" as notifyPeroid`)

Please add your thoughts.

Reply:
follow-up PR, see comment above